File: Selenium/Activities/Actiivty2.java
import org.openqa.selenium.By;
import org.openqa.selenium.WebDriver;
import org.openqa.selenium.WebElement;
import org.openqa.selenium.chrome.ChromeDriver;

public class Activity2 {
    public static void main(String[] args) {
        // Set up the driver
        WebDriver driver = new ChromeDriver();

        // Open the login form page
        driver.get("https://training-support.net/webelements/login-form/");

        // Print the title of the page
        System.out.println("Page title: " + driver.getTitle());

        // Find the username field and enter "admin"
        WebElement username = driver.findElement(By.id("username"));
        username.sendKeys("admin");

        // Find the password field and enter "password"
        WebElement password = driver.findElement(By.id("password"));
        password.sendKeys("password");

        // Find the "Log in" button and click it
        WebElement loginButton = driver.findElement(By.cssSelector("button[type='submit']"));
        loginButton.click();

        // Close the browser
        driver.quit();
    }
}

File: Selenium/Activities/Actiivty2.py
from selenium import webdriver
from selenium.webdriver.common.by import By

# Set up the driver
driver = webdriver.Chrome()

# Open the login form page
driver.get("https://training-support.net/webelements/login-form/")

# Print the title of the page
print("Page title:", driver.title)

# Find the username field and enter "admin"
username = driver.find_element(By.ID, "username")
username.send_keys("admin")

# Find the password field and enter "password"
password = driver.find_element(By.ID, "password")
password.send_keys("password")

# Find the "Log in" button and click it
login_button = driver.find_element(By.CSS_SELECTOR, "button[type='submit']")
login_button.click()

# Close the browser
driver.quit()
